packages/@aws-cdk/aws-apigateway/lib/integration.ts

```diff
@@ -0,0 +1,260 @@
+import iam = require('@aws-cdk/aws-iam');
+import { Method } from './method';
+
+export interface IntegrationOptions {
+    /**
+     * A list of request parameters whose values API Gateway caches.
```

Comment:
This comment is incorrect. It determines request parameters that will make it into the cache key, aka the identity of the request.

Reply:
This is the phrasing in the AWS docs. I added your suffix "It determines request parameters that will make it into the cache key"